presto-docs/src/main/sphinx/release/release-0.233.rst

```diff
@@ -0,0 +1,79 @@
+=============
+Release 0.233
+=============
+
+General Changes
+_______________
+* Fix an optimizer failure introduced in ``0.229``, where a ``LIKE`` pattern is deduced into a constant, e.g., ``col LIKE 'a' and col = 'b'``.
+* Fix ``ArbitraryOutputBuffer`` to avoid skewing output data distribution.
+* Fix function registration for function :func:`classification_fall_out`.
+* Add configuration property ``exchange.async-page-transport-enabled`` to turn on non-blocking IO for page transport,
```

Comment:
My sense is we don't need to add this to our release notes, as it will eventually become the default